templates/zerver/realm_redirect.html

```diff
@@ -28,16 +28,16 @@ <h1 class="get-started">{{ _("Log in to your organization") }}</h1>
                         </div>
                         <div id="errors">
                             {% if form.subdomain.errors %}
-                                {% for error in form.subdomain.errors %}
-                                <div class="alert alert-error">{{ error }}</div>
-                                {% endfor %}
+                            {% for error in form.subdomain.errors %}
+                            <div class="alert alert-error">{{ error }}</div>
+                            {% endfor %}
                             {% endif %}
```

Comment:
This indentation change seems unrelated and less readable?

Reply:
Sorry, I think that was done by mistake. I have reversed the changes in the next commit.